.evergreen.yml

```diff
@@ -4,20 +4,104 @@ ignore:
   - "*.md"
   - ".github/*"
 
+modules:
+- name: evergreen
+  repo: git@github.com:evergreen-ci/evergreen.git
+  branch: master
+  prefix: gopath/src/github.com/evergreen-ci
+
+#######################################
+#              Functions              #
+#######################################
 functions:
   get-project:
     command: git.get_project
     type: setup
     params:
       directory: spruce
 
-  npm-install:
+  get-evergreen-project:
+    command: git.get_project
+    type: setup
+    params:
+      directory: spruce
+      token: ${github_token}
+      revisions:
+        evergreen: ${evergreen_rev}
+
+  setup-mongodb:
```

Comment:
None of these commands will work because there is no makefile. You do need these to run for a working evergreen though

Reply:
they've worked for me dozens of times. the makefile is from the cloned evergreen repo